app/models/miq_request_workflow.rb

```diff
@@ -192,7 +192,12 @@ def validate(values)
         if fld[:required] == true
           # If :required_method is defined let it determine if the field is value
           unless fld[:required_method].nil?
-            fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            Array.wrap(fld[:required_method]).each do |method|
```

Comment:
Up to you but `Array.wrap(nil)` is just `[]` so we could probably take out the `.nil?` check above (and take out another level of indentation :smile:)